token-locker/Cargo.toml

```
@@ -0,0 +1,37 @@
+[package]
+name = "token-locker"
+version = "0.1.0"
+authors = ["Pagoda <hello@pagoda.co>"]
+edition = "2018"
+publish = false
+
+[lib]
+crate-type = ["cdylib", "rlib"]
+
+[profile.release]
+codegen-units = 1
+# Tell `rustc` to optimize for small code size.
+opt-level = "z"
+lto = true
+debug = false
+panic = "abort"
+overflow-checks = true
+
+[dependencies]
+near-sdk = { git = "https://github.com/near/near-sdk-rs", rev = "613adcab6322e4f830efb4d5bde2a810ae642158" }
+near-contract-standards = { git = "https://github.com/near/near-sdk-rs", rev = "613adcab6322e4f830efb4d5bde2a810ae642158" }
+eth-types =  { git = "https://github.com/near/rainbow-bridge", rev = "9ffb08e6613788096f26636f695b4c7e5fa5b261" }
+admin-controlled = { git = "https://github.com/near/rainbow-bridge", rev = "9ffb08e6613788096f26636f695b4c7e5fa5b261" }
```

Let's stick to the recent version of Rainbow Bridge.

```suggestion
eth-types =  { git = "https://github.com/near/rainbow-bridge", rev = "4abc8d9b28d4a9a62b90d35147218a9d08f6721b" }
admin-controlled = { git = "https://github.com/near/rainbow-bridge", rev = "4abc8d9b28d4a9a62b90d35147218a9d08f6721b" }
```

---

see my previous comment